Demos/Blazorise.Demo.Material/wwwroot/css/material/material.css

```diff
@@ -3099,10 +3099,6 @@ fieldset:disabled a.btn { pointer-events: none; }
 
     .custom-control-input.focus ~ .custom-control-label::before, .custom-control-input:active ~ .custom-control-label::before { opacity: 0.12; -webkit-transform: scale(1, 1) translateZ(0); transform: scale(1, 1) translateZ(0); }
 
-    .custom-control-input:checked ~ .custom-control-label::after { color: #ff4081; }
```

Comment:
This should not be removed. Users would still have their own version of material.css so it would not behave the same as in the demo app.